docs/contributing.md

```diff
@@ -3,46 +3,93 @@
 Thank you for being interested in contributing to the `sphinx-book-theme`! You
 are awesome ✨.
 
-This page contains developer documentation to help you get started.
+This project follows the Executable Books Project [contribution guidelines](https://executablebooks.org/en/latest/contributing.html).
+It contains information about our conventions around coding style, pull request workflow, commit messages and more.
 
-```{toctree}
-:hidden:
-contributing-ebp
+This page contains information to help you get started with development on this
+project.
+
+## Development
+
+### Set-up
+
+Get the source code of this project using git:
+
+```bash
+git clone https://github.com/executablebooks/sphinx-book-theme
+cd sphinx-book-theme
 ```
 
-## The EBP Contributing Guide
+To work on this project, you need Python 3.6 or newer. Most of this project's
+development workflow commands use `nox`, which can be installed with pip:
+
+```bash
+pip install nox
+```
 
-This repository follows the `executablebooks/` contributors guide, which
-you can find at [](contributing-ebp). It contains information
+### Building documentation
 
-## Set up your dev environment
+You can now build documentation with your local copy of this theme! Try it
+out on the theme's documentation like so:
 
-To set up your developer environment, do the following:
+```bash
+nox -s docs
+```
 
-* Clone the repository
+This will generate the HTML documentation and compile the relevant stylesheets.
+The generated documentation which can be viewed by opening
+`docs/_build/html/index.html` in your browser.
 
-  ```bash
-  git clone https://github.com/executablebooks/sphinx-book-theme
-  cd sphinx-book-theme
-  ```
-* Install it locally with the developer tools
+### Running Tests
 
-  ```bash
-  pip install -e .[sphinx,testing,code_style]
-  ```
+This theme has a test suite to ensure that all the relevant user content is
+correctly handled. The tests can be run using:
 
-* Install `pre-commit` and activate it for this repository
+```bash
+nox -s tests
```

Comment:
how does this work with arguments passed to pytest? E.g. I often do `pytest -k aspecifictest` or `pytest -s` so that I can start an IPython kernel from within the test.

Reply:
nox just calls pytest pointed at a particular virtualenv that gets created in a folder in the repository foder. But you can still run pytest normally if you don't want to use nox

Reply:
I was wondering if I should mention it here. The form is `nox -s tests-3.8 -- (pytest-args)`.

Reply:
Do we want to document this? If so, please file an issue referencing the above comment.